docs/howto/run_material_mapping.md

```diff
@@ -0,0 +1,119 @@
+
+## Material Validation
+
+Now that the map has been written, you will want to validate it. First you can use the MaterialValidation example. This will perform propagation throughout the detector once it has been decorated with the material map. It will then output material tracks with the same format as the one obtain with the Geantino.
+
+```
+./../build/bin/ActsExampleMaterialValidationDD4hep -n 1000 --mat-input-type file --mat-input-file material-maps.json --output-root true --mat-output-file val-mat-map --dd4hep-input ../Examples/Detectors/DD4hepDetector/compact/OpenDataDetector/OpenDataDetector.xml
+```
+
+To do the validation, five root macro are available in scripts/MaterialMaping :
+
+*  Mat_map.C : General comparison at the track level and 2D map of the detector.
+*  Mat_map_surface_plot.C : for each mapped surface show the position of the material.
+*  Mat_map_surface_plot_ratio.C : material ration between the truth and the validation for each surface.
+*  Mat_map_surface_plot_dist.C : position of the geantino interaction with respect to the surface they are mapped on.
+*  Mat_map_surface_plot_1D.C : 1D distribution of the material in each surface.
+
+```
+mkdir Validation
+
+root -l -b ../acts-framework/scripts/MaterialMaping/Mat_map.C'("propagation-material.root","material-maps_tracks.root","Validation")'
+.q
+
+mkdir Surfaces
+cd Surfaces
+mkdir prop_plot
+mkdir map_plot
+mkdir ratio_plot
+mkdir dist_plot
+mkdir 1D_plot
+cd ..
+
+root -l -b ../acts-framework/scripts/MaterialMaping/Mat_map_surface_plot_ratio.C'("propagation-material.root","material-maps_tracks.root","geometry-map.json",100000,"Surfaces/ratio_plot","Surfaces/prop_plot","Surfaces/map_plot")'
+.q
+root -l -b ../acts-framework/scripts/MaterialMaping/Mat_map_surface_plot_dist.C'("material-maps_tracks.root","geometry-map.json",-1,"Surfaces/dist_plot")'
+.q
+root -l -b ../acts-framework/scripts/MaterialMaping/Mat_map_surface_plot_1D.C'("material-maps_tracks.root","geometry-map.json",100000,"Surfaces/1D_plot")'
+.q
+```
+
+Using the validation plot you can then adapt the binning and the mapped surface to improve the mapping.
+Depending on your root version those macros might not work. They have been tested with version 6.18.04 so you can always revert to that version in case of problem.
+
+## Using a different detector
+
+If you want to use a different detector type of detector you will first need to ensure that the relevant packages were added during the compilation. After that if your detector is a DD4hep detector you will just need to replace the path given to the '--dd4hep-input' option. In case it is another type of detector implementation you can replace DD4hep in the name of the algorithm by what correspond to your detector implementation. For more information on how to include your detector in that case you can refer to the documentation of the algorithm using the `-h` option.
```

Comment:
```suggestion
If you want to use a different type of detector, you will first need to ensure that the relevant packages were added during the compilation. After that, if your detector is a DD4hep detector you will just need to replace the path given to the '--dd4hep-input' option. In case it is another type of detector implementation, you can replace DD4hep in the name of the algorithm by what corresponds to your detector implementation. For more information on how to include your detector in that case you can refer to the documentation of the algorithm using the `-h` option.
```